drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2757,14 +2759,19 @@ void RasterizerSceneGLES3::_render_list_template(RenderListParameters *p_params,
 				// Invalidate the previous index array
 				prev_index_array_gl = 0;
 			}
-
+			
 			bool use_index_buffer = index_array_gl != 0;
-			if (prev_index_array_gl != index_array_gl) {
-				if (index_array_gl != 0) {
-					// Bind index each time so we can use LODs
-					glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, index_array_gl);
+
+			if (use_wireframe) {				
+				glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, wireframe_index_array_gl);
+			} else {
```

Comment:
I think it might be a little cleaner to do the following instead:
```
if (p_params->force_wireframe) {
    GLuint wireframe_index_array_gl = mesh_storage->mesh_surface_get_index_buffer_wireframe(mesh_surface);
    index_array_gl = wireframe_index_array_gl? wireframe_index_array_gl : index_array_gl;
}
```
This way we can keep the previous logic and only rebind when necessary. And we only have to lookup the  wireframe index array when forcing wireframe mode